transformers4rec/torch/utils/data_utils.py

```diff
@@ -367,6 +368,34 @@ def __init__(
         self.schema = schema
         self.max_sequence_length = max_sequence_length
 
+    @staticmethod
+    def _get_pad_fn(padding_lengths):
+        def pad_fn(x, y):
+            new_x = pad_batch(x, padding_lengths)
+            new_y = pad_batch(y, padding_lengths)
+            return new_x, new_y
+
+        return pad_fn
+
+    @staticmethod
+    def _augment_schema(
+        schema,
+        cats=None,
+        conts=None,
+        labels=None,
+    ):
+        schema = schema.select_by_name(conts + cats + labels)
```

Comment:
Maybe we should remove the `=None` for cats, conts & labels?

Reply:
that's clearer without the default now. it wouldn't have worked as None before this change either.

Reply:
On second thought, I've put the None back and added support for the None to the method by setting to an empty list if not provided. 

It doesn't appear to be captured by any tests, but this would enable training a model with only continuous or only categorical features. Currently you need to have at least one of each for the current version of this MerlinDataloader to work.